boards: arm: stm32f469i_disco: add sdmmc1

Desvauxm-st opened the pull request:
adds sdmmc1 node on stm32f469i_disco board
to use SDIO bus for SD card access
the mandatory clock is 48 Mhz.

Desvauxm-st committed:
boards: arm: stm32f469i_disco: add sdmmc1

adds sdmmc1 node on stm32f469i_disco board
to use SDIO bus for SD card access
the mandatory clock is 48 Mhz.

Signed-off-by: Marc Desvaux <marc.desvaux-ext@st.com>

erwango reviewed (changes requested):
Please update board documentation

Desvauxm-st commented:
Yes

Desvauxm-st committed:
boards: arm: stm32f469i_disco: doc: add SDIO

adds SDIO(sdmmc1) node on stm32f469i_disco board
to use SDIO bus for SD card access

Signed-off-by: Marc Desvaux <marc.desvaux-ext@st.com>
